src/views/router_views/individualPscList/individualPscList.njk

```diff
@@ -0,0 +1,43 @@
+{% extends "layouts/default.njk" %}
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+{% set title = i18n.individual_psc_list_main_title %}
+{% set dummyRadioButtons = i18n.dummyRadioButtons %}
+{% set dummyPscOption = i18n.dummyPscOption %}
+
+{% block main_content %}
+    <div class="govuk-grid-row">
+
+            <h1 class="govuk-fieldset__legend--l">{{ title }}</h1>
+            <h3 class="govuk-hint">{{ i18n.individual_psc_list_hint }}</h3>
+
+            {{ govukRadios({
+              name: "dummyRadioButtons",
+              fieldset: {
+                legend: {
+                  text: dummyRadioButtons,
+                  isPageHeading: true,
+                  classes: "govuk-fieldset__legend--l"
+                }
+              },
```

Comment:
If you utilise the  nunjunks macro options instead you can do something like this:
{{ govukRadios({
              name: "pscList",
              fieldset: {
                legend: {
                  text: i18n.individual_psc_list_main_title,
                  isPageHeading: true,
                  classes: "govuk-fieldset__legend--l"
                }
              },
              hint: { 
                    text: i18n.individual_psc_list_hint
                  },
              items: [